create/config.go

```diff
@@ -0,0 +1,151 @@
+package create
+
+import (
+	"bufio"
+	"cfctl/common"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+type CreateConfig struct {
+	Context string
+}
+
+var CreateConfigVar CreateConfig
+
+// Create context given by argument with user input
+func AddContext(cmd *cobra.Command, args []string) {
+	var ConfigFileContents common.CLIConfig
+
+	defaultUrl := "http://localhost:8080/api"
+
+	configFile, err := common.GetConfigFilePath()
+	if err != nil {
+		os.Exit(1)
+	}
+
+	// Create file if doesn't exist
+	f, err := os.OpenFile(configFile, os.O_CREATE|os.O_RDWR, 0644)
+	if err != nil {
+		os.Exit(1)
+	}
+	defer f.Close()
+	// Get the file's content
+	content, err := io.ReadAll(f)
+	if err != nil {
+		os.Exit(1)
+	}
+	err = yaml.Unmarshal(content, &ConfigFileContents)
+	if err != nil {
+		os.Exit(1)
+	}
+
+	ConfigFileContents.Default = getUserInputDefaultContext(ConfigFileContents.Default)
+
+	inputUrl, err := getUserInput("Enter url: ")
+	if inputUrl == "" || err != nil {
+		fmt.Printf("Using default URL %v\n", defaultUrl)
+		inputUrl = defaultUrl
+	}
+
+	ConfigFileContents = createContext(ConfigFileContents, inputUrl, CreateConfigVar.Context)
+
+	d, err := yaml.Marshal(ConfigFileContents)
+	if err != nil {
+		fmt.Printf("Could not marshal contents %v\n", ConfigFileContents)
+		os.Exit(1)
+	}
+
+	// Overwrite the file with the new contents
+	f.Truncate(0)
+	f.Seek(0, 0)
+	f.Write(d)
+	fmt.Printf("Successfully wrote new context for '%v'\n", CreateConfigVar.Context)
+}
+
+// Output the config, by context if specified
```

Comment:
```suggestion
// GetContext: outputs the config, by context if specified
```

Any function annotations I think have to start with the name

Reply:
Ah ok - thank you. Amended elsewhere!